compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -167,8 +167,10 @@ message ScenarioError {
 
   message EmptyContractKeyMaintainers {
     Identifier template_id = 1;
-    Value arg = 2;
-    Value key = 3;
+    Value key = 2;
+    // Optional location of the update expression leading
+    // to the violation.
+    Location location = 3;
```

Comment:
What makes this one special? We don’t have locations for any of the other errors afaik.

Reply:
And afaik we also always include the last location anyway so I’m not sure when this provides extra information.

Reply:
dropped.